src/cloudant/design_document.py

```diff
@@ -49,6 +49,36 @@ def __init__(self, database, document_id=None):
             self.setdefault(prop, dict())
 
     @property
+    def validate_doc_update(self):
+        """
+        Provides an accessor property to the update validators dictionary in
+        the locally cached DesignDocument.  Update validators evaluate whether a
+        document should be written to disk when insertions and updates are attempted.
+
+        Note: Update validators do not require a query because they implicitly
```

Comment:
I don't think we should say this. It makes it sound like accessing the property runs the validator.